packages/subscription-manager/src/components/SubscriptionManagerContainer/SubscriptionManagerContainer.tsx

```diff
@@ -16,18 +17,25 @@ const SubscriptionManagerContainer = ( { children }: SubscriptionManagerContaine
 	const translate = useTranslate();
 
 	return (
-		<Main className="subscription-manager-container">
-			<DocumentHead title="Subscriptions" />
-			<FormattedHeader
-				brandFont
-				headerText={ translate( 'Subscription management' ) }
-				subHeaderText={ translate(
-					'Manage your WordPress.com newsletter and blog subscriptions.'
-				) }
-				align="left"
+		<>
+			<UniversalNavbarHeader
+				className="subscription-manager-header"
+				variant="minimal"
+				isLoggedIn={ false }
 			/>
-			{ children }
-		</Main>
+			<Main className="subscription-manager-container">
+				<DocumentHead title="Subscriptions" />
+				<FormattedHeader
+					brandFont
+					headerText={ translate( 'Subscription management' ) }
```

Comment:
> Are these strings finalized?

Yes, the following are the designs: inDLaEQV8jJ21O4WXawIsJ-fi-712%3A13475

>  if we don't want to translate them just yet because they aren't final, we can set translate() to be an identity ( x ) => x function while they are being finalized.

Thank you, @tyxla ! This is something I did not really think about, but this is a great solution for such case.  I will keep this in mind for future development in Calypso.